### PR TITLE
CDAP-11491 Removing Standalone and SDK from docs

### PR DESCRIPTION
**Pull request by awholegunch**
Running as a Quick Build: https://builds.cask.co/browse/CDAP-DQB351-1

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -48,8 +48,8 @@ transform, and load) pipelines all the way up to complex, enterprise-scale data-
 applications.
 
 Developers can build and test their applications end-to-end in a full-stack, single-node
-installation. CDAP can be run either standalone, deployed within the Enterprise or hosted
-in the Cloud.
+installation. CDAP can be run either as a Local Sandbox, deployed within the Enterprise or
```
Comment (bdmogal) [inline README.rst]: deployed within the Enterprise on-premise?

File: cdap-docs/developers-manual/source/building-blocks/core.rst [hunk 2]
```diff
@@ -18,15 +18,15 @@ low-level APIs.
...
+while providing access to many useful and powerful services provided by CDAP such as
 distributed transactions, service discovery, and the ability to dynamically scale
 processing units.
 
 Applications are abstracted away from the platform that runs the application. When you
 deploy and run the application into a specific installation of CDAP, the appropriate
 implementations of all services and program runtimes are injected by CDAP; the application
 does not need to change based on the environment. This allows you develop applications in
-one environment—such as on your laptop using a standalone CDAP for testing—and then
+one environment—such as on your laptop using a CDAP Sandbox for testing—and then
```
Comment (bdmogal) [inline cdap-docs/developers-manual/source/building-blocks/core.rst]: Local or Cloud Sandbox? Do we mention somewhere that CDAP Sandbox could mean Local or Cloud Sandbox?

File: cdap-examples/README.rst [hunk 3]
```diff
@@ -42,7 +42,12 @@ CountRandom
 DataCleansing
 -------------
 - Demonstrates incrementally consuming partitions of a partitioned fileset using MapReduce.
-      
+
+Decision Tree Regression
```
Comment (bdmogal) [inline cdap-examples/README.rst:46]: this is unrelated right?

**Review by bdmogal — APPROVED** [review]
@awholegunch I've taken a look. Mostly LGTM. However, this PR has a whole lot of unrelated changes. Did your editor config change? If not, can you please split them up the next time, especially so close to the release? Want to make sure we don't miss the important stuff.

File: README.rst (hunk 1 above)
Comment (awholegunch) [reply README.rst]: Added "on-premises".

File: cdap-docs/developers-manual/source/building-blocks/core.rst (hunk 2 above)
Comment (awholegunch) [reply cdap-docs/developers-manual/source/building-blocks/core.rst]: We don't talk (yet) about "Cloud Sandbox", so I'm thinking this should be (for now) "CDAP Local Sandbox". Re-wrote sentence to correct a number of errors.

File: cdap-examples/README.rst (hunk 3 above)
Comment (awholegunch) [reply cdap-examples/README.rst:46]: Well, yes…

**Comment by awholegunch** [comment]
@bdmogal At Ali's suggestion, I changed my editor config a while ago to remove trailing whitespace, in keeping with our Java file standards and so I don't pollute them when editing. This has the side affect of cleaning up any files that I touch. Probably the only way to prevent this additional editing is to do a pass over all documentation files, and remove any trailing spaces.

**Comment by awholegunch** [comment]
Re-ruuning Quick Build: https://builds.cask.co/browse/CDAP-DQB351-2

**Comment by awholegunch** [comment]
Re-running Quick Build: https://builds.cask.co/browse/CDAP-DQB351-3 (passes)
